http/codegen/openapi/v2/builder.go

```diff
@@ -639,7 +639,7 @@ func buildPathFromExpr(s *V2, root *expr.RootExpr, h *expr.HostExpr, route *expr
 			}
 			requirements[i] = requirement
 		}
-
+		_, is_deprecated := endpoint.MethodExpr.Meta.Last("openapi:deprecated")
```

Comment:
nitpick :) [idiomatic Go](https://go.dev/doc/effective_go#names) does not use underscores in variable names so maybe just `deprecated`? That would also be consistent with the field name.